src/runtime_src/driver/xclng/drm/xocl/mgmtpf/mgmt-core.c

```diff
@@ -863,6 +878,8 @@ static int xclmgmt_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 		return rc;
 	}
 
+	pci_set_master(pdev);
```

Comment:
Why do we need to set it explicitly now? Does it change anything for XDMA?

Reply:
Bus Master is required before configuring MSIX